forte/data/extractor/base_extractor.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from abc import ABC
+from typing import Dict, Any, Union, Type
+from ft.onto.base_ontology import Annotation
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.data.vocabulary import Vocabulary
+from forte.data.converter.feature import Feature
+
+
+class WrapperClass:
```

Comment:
I am still concerned about using `WrapperClass`. One issue I found is file format. Since `WrapperClass` is used, some methods cannot be referenced and the format tool will show error. Not sure what could be a better way to do this.

Reply:
I will removed wrapper class and wrap the function normally.